### PR TITLE
imagehash feedstock

### PR DESCRIPTION
**Pull request by bjlittle**
New `imagehash` feedstock

### CODE REVIEW

File: recipes/imagehash/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set version = "3.1" %}
+
+package:
+  name: imagehash
+  version: {{ version }}
+
+source:
+  fn: imagehash-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/0c/1c/b6f539cc08cf8cfb12cb59a6883ee7f1cc0937cf3709247c2a15c8a1e665/ImageHash-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/imagehash/meta.yaml]: Can you change to the more readable `https://pypi.io/packages/source/I/ImageHash/ImageHash-{{ version }}.tar.gz`?

File: recipes/imagehash/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
```
Comment (ocefpaf) [inline recipes/imagehash/meta.yaml]: I am not sure it needs `numpy x.x`. I could not find `numpy.get_include()` anywhere.

File: recipes/imagehash/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  license: BSD 2-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'A Python Perceptual Image Hahsing Module'
+
+extra:
+  recipe-maintainers:
+    - JohannesBuchner
+    - bjlittle
+
```
Comment (ocefpaf) [inline recipes/imagehash/meta.yaml]: Feel free to add me too.

**Comment by bjlittle** [comment]
@ocefpaf :smile:

**Pull request merged**